### PR TITLE
Ragini/sidebar improvements/test

### PR DESCRIPTION
**Pull request by ViaSocket-Git**

### CODE REVIEW

File: src/components/main/sidebar.scss [hunk 1]
```diff
@@ -143,7 +143,7 @@ body .sidebar {
   }
 
   .primary-sidebar {
-    width: 100%;
+    width: 100% !important;
```
Comment (Saurabh186) [inline src/components/main/sidebar.scss:146]: use bootstrap claas by default important lag ke ata hai

File: src/components/main/sidebar.scss [hunk 2]
```diff
@@ -1104,3 +1104,49 @@ input[type='text']:focus {
   color: rgba(128, 128, 128, 0.7);
   font-size: 12px;
 }
+
+//Internal Page Sidebar
+.sidebar {
+  background-color: $secondary-color !important;
```
Comment (Saurabh186) [inline src/components/main/sidebar.scss:1110]: remove important and check output

File: src/components/styles.scss [hunk 3]
```diff
@@ -520,7 +520,7 @@ button.btn {
 
 .custom-raw-editor {
   width: 100% !important;
-  height: 250px !important;
+  // height: 250px !important;
```
Comment (Saurabh186) [inline src/components/styles.scss:523]: Are you sure? Also remove the commented code

**Review by Saurabh186 — CHANGES_REQUESTED** [review]

File: src/components/main/main.scss [hunk 4]
```diff
@@ -4301,3 +4301,25 @@ a {
...
   }
 }
+
+@media screen and (max-width: 768px) {
+  .content-header {
+    margin-left: 4rem !important;
+  }
+}
+
+.table-container {
```
[the remark this answers is not in the document]
Comment (ViaSocket-Git) [reply src/components/main/main.scss]: Resolved

File: src/components/main/sidebar.scss (hunk 1 above)
Comment (ViaSocket-Git) [reply src/components/main/sidebar.scss:146]: Resolved

File: src/components/styles.scss (hunk 3 above)
Comment (ViaSocket-Git) [reply src/components/styles.scss:523]: Resolved

File: src/components/main/sidebar.scss (hunk 2 above)
Comment (ViaSocket-Git) [reply src/components/main/sidebar.scss]: Resolved

**Review by Saurabh186 — APPROVED** [review]

**Pull request merged**